test/library/packages/Zarr/EXECENV

```diff
@@ -1 +1,3 @@
+#!/bin/bash
 LD_LIBRARY_PATH=.
+DYLD_LIBRARY_PATH=.
```

Comment:
I don't have a problem with this approach for the test itself, but generally speaking, there are better approaches to this kind of thing (e.g. using `-Wl,-rpath` as in PR #23590)

Reply:
I'll keep this in mind for future testing.